libraries/cmd/carrier.cpp

```diff
@@ -245,12 +245,12 @@ void Carrier::EjectCargo(unsigned int index) {
             }
             float arot = 0;
             const float grot =
-                    configuration().graphics.generic_cargo_rotation_speed * 3.1415926536 / 180;
+                    configuration().graphics.generic_cargo_rotation_speed_flt * 3.1415926536F / 180.0F;
```

Comment:
these should get changed over to `M_PI` which we're already using elsewhere in the code, such as in `libraries/cmd/unit_generic.cpp`

Reply:
The only issue I see with this is that `M_PI` is a `double`, not a `float`. If I use it, the whole calculation on each of these lines will become a `double` calculation. Not the end of the world, but worth pointing out, I think.

Reply:
I believe this is also taken care of.